src/mx/_impl/mx_javamodules.py

```diff
@@ -1085,14 +1088,23 @@ def safe_path_arg(p):
                     # The --system=none and --limit-modules options are used to support distribution defined modules
                     # that override non-upgradeable modules in the source JDK (e.g. org.graalvm.sdk is part of a
                     # GraalVM JDK). This means --module-path needs to contain the jmods for the JDK modules.
-                    javac_args.append('--system=none')
+                    if _jlink_libraries():
```

Comment:
Can you please refactor the result of `_jlink_libraries()` into a local variable.

Reply:
This is no longer present in the latest version.